scenarios/Loadfilename_as_column_s3/README.md

```diff
@@ -0,0 +1,58 @@
+# Workflow: Scenario (Load file name as a column from S3)
+
+## Scenario
+
+The purpose of this scenario is to capture loaded file name as data from S3 on AWS.
+
+
+*Steps*
+1. Obtain file names and file contents from multiple files in a specific bucket of S3.
+   the file sample
+   name: files_aaa_20210401
+   contents:
```

Comment:
L10 - L12 letters are in the same line. Is this intentional?

Reply:
Thank you for your check! There need to line break there. I add the line break.